paystell-frontend/src/components/ui/input.stories.tsx

```
@@ -0,0 +1,168 @@
+    placeholder: "Enter your email",
+  },
+};
+
+export const Password: Story = {
+  args: {
+    type: "password",
+    placeholder: "Enter your password",
+  },
+};
+
+export const Number: Story = {
```

Biome (2.1.2)

[error] Do not shadow the global "Number" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)